library/alloc/tests/slice.rs

```diff
@@ -978,7 +978,7 @@ fn test_chunksator_0() {
 }
 
 #[test]
-fn test_chunks_exactator() {
+fn test_chunks_extractor() {
```

Comment:
We're not extracting anything -- this is supposed to be the `chunks_exact` variation of `chunksator` above. All the "*ator" variations are definitely weird though...

Reply:
I've changed everything to spell out `iterator` see https://github.com/rust-lang/rust/compare/6215b33a6031c265f5e0646eb1531efc8c0ec8f9..9d2d40d2640c7cf9f328d035099e209b6c99b108